src/reducers/providers.js

```diff
@@ -37,7 +37,7 @@ function providers(state = defaultState, action) {
       linkProvider: { isLinking: false, linkStatus: 'success' }
     };
   case types.OAUTH_CALLBACK_SUCCESS:
-    window.location.assign(action.redirectUri);
+    //window.location.assign(`/dashboard/profiles/${action.profileId}`);
```

Comment:
Was this meant to be commented out?

Reply:
in reality it could be removed, I'll do that